### ISSUE TITLE
brew install osgeo-gdal failed  Error - No available formula with the name "cryptopp"

### ISSUE THREAD

**Issue by vishalsynerzip** [OSGeo/homebrew-osgeo4mac#1394]
**Please answers the following**

I've... 
- I read
	- [x] the [reporting issues](https://osgeo.github.io/homebrew-osgeo4mac/docs/reporting-issues/)
	- [x] the [troubleshooting](https://osgeo.github.io/homebrew-osgeo4mac/docs/troubleshooting/)
- [x] I checked the documentation and found no answer
- [x] I tried to install after following the suggestions
- [x] I am running the latest version (`brew update` twice?)
- [x] I checked to make sure that this issue has not already been filed
- [ ] I uploaded logs: `brew gist-logs formula`

**Describe the bug**
While installing osgeo-dal the error gets thrown 
Error: No available formula with the name "cryptopp" (dependency of osgeo/osgeo4mac/osgeo-gdal)

**To Reproduce**

1. Go to Terminal
2. Type brew tap osgeo/osgeo4mac then brew install osgeo-gdal
3. Scroll down on Terminal

**Expected behavior**

The osgeo-gdal would get installed.

**Screenshots**

**Additional context**
I am running this on the Catalina macOS version - 10.15.6

`brew doctor` output: Your system is ready to brew.

```shell
==> Installing osgeo-gdal from osgeo/osgeo4mac
Error: No available formula with the name "cryptopp" (dependency of osgeo/osgeo4mac/osgeo-gdal)
==> Searching for a previously deleted formula (in the last month)...
cryptopp was deleted from homebrew/core in commit f5961382fd:
  cryptopp: delete formula.
  This has been without a bottle for a long time and it doesn't seem
  possible to fix. We're a binary package manager so delete it.

To show the formula before removal run:
  git -C "$(brew --repo homebrew/core)" show f5961382fd^:Formula/cryptopp.rb

If you still use this formula consider creating your own tap:
  https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

```

**Logs**

**Comment by alazarolop** [comment]
Hi @vishalsynerzip , thank your for posting. I'm aware of this issue. The formula isn't getting built, I don't know if related to this dependency or another bug, but I haven't been able to fix it in the bottles brunch. The log is available [here](https://circleci.com/api/v1.1/project/github/OSGeo/homebrew-osgeo4mac/6978/output/107/0?file=true&allocation-id=5f3f7cbeedc3d807adcf4be8-0-build%2F6373F69)

The only error I detect
```
jpeg2000dataset.cpp:611:28: error: assigning to 'char *' from incompatible type 'const char *'
    if ( !(pszFormatName = jas_image_fmttostr( iFormat )) )
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [../o/jpeg2000dataset.lo] Error 1
make[1]: *** [jpeg2000-install-obj] Error 2
make: *** [frmts-target] Error 2
```

I need to invest more time. Help would be more than welcome.

**Comment by alazarolop** [comment]
It should be fixed with 221f6e21ffd0ea3108d9e7a686eeee3a60e309dc

**Comment by vishalsynerzip** [comment]
Thanks @alazarolop ... I have tested it and it is working fine.
